packages/playground/blueprints/src/lib/steps/run-sql-file.ts

```diff
@@ -0,0 +1,48 @@
+import { StepHandler } from '.';
+
+/**
+ * @inheritDoc runSqlFile
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ * 	    "step": "runSqlFile",
+ * 		"path": "wordpress.org/plugins"
```

Comment:
is this a valid path? would be nice if we used something that doesn't also reference a website that returns HTML

Reply:
@dmsnell  This file has been removed.

https://github.com/WordPress/wordpress-playground/pull/745/files